deeprank2/neuralnets/gnn/ginet.py

```diff
@@ -10,7 +10,16 @@
 # ruff: noqa: ANN001, ANN201
 
 
-class GINetConvLayer(nn.Module):  # noqa: D101
+class GINetConvLayer(nn.Module):
+    """GiNet convolutional layer for graph neural networks.
+
+    Args:
+        in_channels: Number of input features.
+        out_channels: Number of output features.
+        number_edge_features: Number of edge features. Defaults to 1.
+        bias: If set to :obj:`False`, the layer will not learn an additive bias. Defaults to False.
```

Comment:
We've consistently been just using True/False throughout the docstrings, without backticks or using :obj:

```suggestion
        bias: If set to False, the layer will not learn an additive bias. Defaults to False.
```